lua/neogit/popups/cherry_pick/init.lua

```diff
@@ -0,0 +1,40 @@
+local popup = require("neogit.lib.popup")
+local actions = require("neogit.popups.cherry_pick.actions")
+
+local M = {}
+
+function M.create(env)
+  local in_progress = actions.pick_or_revert_in_progress()
+
+  local p = popup
+    .builder()
+    :name("NeogitCherryPickPopup")
+    :switch("F", "ff", "Attempt fast-forward", { enabled = true })
+    -- :switch("x", "x", "Reference cherry in commit message", { cli_prefix = "-" })
+    -- :switch("e", "edit", "Edit commit messages", false)
+    -- :switch("s", "signoff", "Add Signed-off-by lines", false)
+    -- :option("m", "mainline", "", "Replay merge relative to parent")
+    -- :option("s", "strategy", "", "Strategy")
+    -- :option("S", "gpg-sign", "", "Sign using gpg")
+    :group_heading_if(not in_progress, "Apply here")
+    :action_if(not in_progress, "A", "Pick", actions.pick)
+    :action_if(not in_progress, "a", "Apply", actions.apply)
+    :action_if(not in_progress, "h", "Harvest")
+    :action_if(not in_progress, "m", "Squash")
+    :new_action_group_if(not in_progress, "Apply elsewhere")
+    :action_if(not in_progress, "d", "Donate")
+    :action_if(not in_progress, "n", "Spinout")
```

Comment:
I suppose these will be filled in at a later point in time

Reply:
Reference to nomenclature: https://magit.vc/manual/magit/Cherry-Picking.html#index-A-h

Reply:
Yeah.. i've tried to make sense of how these are implemented in Magit and.. I'm putting it off for now. Maybe someone beats me to it 😅 But for the sake of previewing whats to come, and layout, I added them anyways.